CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration_TemplateMod.uc

```diff
@@ -1126,6 +1126,57 @@ static function PatchDoomRemovalCovertAction()
 	ActionTemplate.bMultiplesAllowed = false;
 }
 
+static function PatchUniqueCovertActions ()
+{
+	RemoveUniqueFromCovertAction('CovertAction_SuperiorPCS');
+	RemoveUniqueFromCovertAction('CovertAction_SuperiorWeaponUpgrade');
+}
+
+static function RemoveUniqueFromCovertAction (name ActionName)
+{
+	local X2StrategyElementTemplateManager TemplateManager;
+	local X2CovertActionTemplate ActionTemplate;
+	
+	TemplateManager = class'X2StrategyElementTemplateManager'.static.GetStrategyElementTemplateManager();
+
+	ActionTemplate = X2CovertActionTemplate(TemplateManager.FindStrategyElementTemplate(ActionName));
+	
+	ActionTemplate.bMultiplesAllowed = false;
+	ActionTemplate.bUnique = false;
+}
+
+static function PatchResourceGatheringCovertActions ()
+{
+	local X2StrategyElementTemplateManager TemplateManager;
+	local X2CovertActionTemplate ActionTemplate;
+	
+	TemplateManager = class'X2StrategyElementTemplateManager'.static.GetStrategyElementTemplateManager();
+
+	CreateLategameCovertAction('CovertAction_GatherSupplies', 'CovertActionEngineerStaffSlot');
+	CreateLategameCovertAction('CovertAction_GatherIntel', 'CovertActionScientistStaffSlot');
+}
+
+static function CreateLategameCovertAction (name ActionName, name OptionalSlotStaff)
```

Comment:
For a method called `Create[...]` i see an alarming lack of new things being... created. 😄

Rename to something like `ConvertActionToLateGame`